content/docs/ops/configuration-management.md

```diff
@@ -29,21 +29,21 @@ Security tests need to be executed in the staging environment where changes are
 
 ## Change workflow
 
-1. All configuration changes must flow through a git repository, centrally managed through GitHub, unless they contain sensitive information. In these cases, sensitive information should be stored in an S3 bucket with a proper security policy, encryption, and versioned, so that changes can be easily rolled back.
+1. All configuration changes must flow through a git repository, centrally managed through GitHub, unless they contain sensitive information. In these cases, sensitive information should be stored in an S3 bucket with a proper security policy and encryption, versioned such that changes can be easily rolled back.
```

Comment:
Not sure whether this makes the policy more clear - depends on how S3 works, which I'm not deeply familiar with, so I'll leave that bit to others to 👍 ! Rest looks good to me.